azure/v29.0.1/README.md

```diff
@@ -0,0 +1,32 @@
+# :zap: Giant Swarm Release v29.0.1 for Azure :zap:
+
+## Changes compared to v29.0.0
+
+### Apps
+
+- coredns from v1.21.0 to v1.22.0
+- cert-exporter from v2.9.1 to v2.9.2
+- observability-bundle from v1.5.1 to v1.6.1
```

Comment:
I remember that it was mentioned that we don't want to bump a minor version in a patch release, but is that so?

Reply:
from version is wrong
```suggestion
- observability-bundle from v1.5.3 to v1.6.1
```